lib/fs.js

```diff
@@ -54,7 +54,7 @@ const {
 
 const { FSReqCallback, statValues } = binding;
 const internalFS = require('internal/fs/utils');
-const { getPathFromURL } = require('internal/url');
+const { addFeatureDetection, getPathFromURL } = require('internal/url');
```

Comment:
`addFeatureDetection` is part of `internal/util` not `internal/url`.

Reply:
bah... I BROKE IT